### DISCUSSION TITLE
Selecting Multiple Items

### DISCUSSION THREAD

**Discussion by CamdenGO** [GrandOrgue/grandorgue#1417] [Help]
What if in Audio/Midi> Organ Settings, we were able to select multiple items sequentially. Such as, if want to select Montre Pipe 001, and Montre Pipe 002, I could hold down the left click and drag the mouse over the two pipes and then I could adjust amplitude, tuning, etc. 
That would affect those two pipes. 
Just an idea, thoughts?

**Comment by oleg68** [comment]
This feature is already present. You can select multiple oblects in OrganSettings with LCllick - Shift+LCllick/Ctrl+LClick
